lib/api_repo.js

```diff
@@ -573,32 +574,44 @@ ApiRepo.prototype._buildProtos =
          * @param {?Error} err Failure if specified.
          * @param {string[]} allProtos The list of all proto files.
          */
-        var makeNodeModule = function makeNodeModule(err, allProtos) {
+        var makeNodeModule = function makeNodeModule(err) {
           if (err !== null) {
             findOutputs(err);
             return;
           }
-          var fullPathProtos = _.map(allProtos, function(x) {
-            return path.join(this.repoDir, x);
-          }.bind(this));
 
-          var includePath = _.union(this.includePath, [this.repoDir]);
-          if (!this.opts.nodejsUsePbjs) {
-            makeProtoBasedNodeModule(fullPathProtos, includePath);
-            return;
+          var that = this;
+          function makeModule(err) {
+            var includePath = _.union(that.includePath, that.repoDirs);
+            if (!that.opts.nodejsUsePbjs) {
+              makeProtoBasedNodeModule(fullPathProtos, includePath);
+              return;
+            }
+            var opts = {
+              root: that.repoDir,
```

Comment:
Fixed.

(I think we don't have tests for `nodejsUsePbjs=false` -- I think this is why this was not caught by the tests.)